src/articles.ts

```diff
@@ -1,13 +1,13 @@
-import { Iri, JsonLdObj } from 'jsonld/jsonld-spec';
+import { BlankNode, DatasetCore } from 'rdf-js';
 
-interface Articles extends Iterable<JsonLdObj> {
-  add(article: JsonLdObj): Promise<void>;
+interface Articles extends Iterable<[BlankNode, DatasetCore]> {
```

Comment:
`BlankNode` will be replaced with `NamedNode` in the future.

Reply:
Elaborate with an example on the difference?

Reply:
e.g. does the `NamedNode` have a URI?

Reply:
`NamedNode` is an IRI (=unicode URI). Since we're using Hydra and declare it as a Hydra resource, that means it would also be dereferenceable (ie _also_ a URL). `BlankNode` is just a local identifier, in this case inside the article list.